src/SDKs/_metadata/signalr_resource-manager.txt

```diff
@@ -1,14 +1,14 @@
 ﻿Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/signalr/resource-manager/readme.md --csharp --csharp-sdks-folder=C:\azure-sdk-for-net\tools\..\src\SDKs\ --version=latest --reflect-api-versions
-2018-05-03 17:51:57 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/signalr/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\Code\azure-sdk-for-net\tools\..\src\SDKs\
+2018-08-31 09:44:19 UTC
 1) azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      master
-Commit:      0a68675c56fceef8b5069a3026286539356feadb
```

Comment:
Please run msbuild build.proj and regenerate the code

Reply:
Regenerated with

```
cd tools
.\generateTool.ps1 -ResourceProvider signalr/resource-manager -PowershellInvoker
```

It seems the `generateTool.ps1` and `Start-CodeGeneration` in `tools\SdkBuildTools\psModules\CodeGenerationModules\generateDotNetSdkCode.psm1` are not passing the `ResourceProvider` parameter correctly. I need to manually update the PowerShell code to make it work.